### PR TITLE
Leland rule feed initial 5

### PR DESCRIPTION
**Pull request by enncoded**

### CODE REVIEW

File: javascript/ecdsa/security/audit/ecdsa-signature-forgery.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+rules:
+  - id: ecdsa-signature-forgery
+    patterns:
+      - pattern-either:
+          - pattern-inside: |
```
Comment (kurt-r2c) [inline javascript/ecdsa/security/audit/ecdsa-signature-forgery.yaml:5]: Are these meant to be two separate `pattern-inside` rules?  This reads to me like you could potentially have:

```
var SomeLib = require('definitely-not-starkbank-ecdsa')
var ECDSA =  SomeLib.Ecdsa
ECDSA.verify(someArgs)
```

Which would then match as long as the `starkbank-ecdsa` library was present somewhere in the project

File: javascript/node-fetch/security/audit/node-fetch-cookie-leak.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    patterns:
+      - pattern-either:
+        - pattern: |
+            fetch(..., { ..., headers: {..., 'Cookie': ..., ...}, ... })
+        - pattern: |
+            const $HEADERS = {..., 'Cookie': ..., ...}
+            ...
+            fetch(..., { headers: $HEADERS }, ...)
+        - pattern: |
+            const $OPTIONS = {..., headers: $HEADERS, ...}
```
Comment (kurt-r2c) [inline javascript/node-fetch/security/audit/node-fetch-cookie-leak.yaml:12]: this case doesn't guarantee cookies are sent, does it?  While $HEADERS might be set in one of the other `pattern-either` definitions, this pattern can match `options` with no `Cookie` key.  We should add a test case for same.

**Review by kurt-r2c — CHANGES_REQUESTED** [review]

File: javascript/ecdsa/security/audit/ecdsa-signature-forgery.yaml (hunk 1 above)
Comment (enncoded) [reply javascript/ecdsa/security/audit/ecdsa-signature-forgery.yaml:5]: Good catch! I'll fix that

File: javascript/node-fetch/security/audit/node-fetch-cookie-leak.yaml (hunk 2 above)
Comment (enncoded) [reply javascript/node-fetch/security/audit/node-fetch-cookie-leak.yaml:12]: Fixed! I'll send you the updated rule and test on slack

**Pull request closed without merging**